iroha/src/bridge.rs

```diff
@@ -1 +1,93 @@
 //! This module contains functionality related to `Bridge`.
+
+use crate::prelude::*;
+use iroha_derive::Io;
+use parity_scale_codec::{Decode, Encode};
+use std::fmt::Debug;
+
+/// Enumeration of all supported bridge kinds (types). Each variant represents some
+/// distributed ecosystem or protocol which can be used within Iroha.
+#[derive(Encode, Decode, PartialEq, Eq, Debug, Clone, Copy, Hash, Io)]
+pub enum BridgeKind {
+    /// [Polkadot](https://polkadot.network/) network.
+    Polkadot,
+}
+
+/// Identification of a Bridge definition. Consists of Bridge's kind and name.
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Encode, Decode)]
+pub struct BridgeDefinitionId {
+    /// Bridge's kind (type).
+    pub kind: BridgeKind,
+    /// Bridge's name.
+    pub name: String,
+}
+
+impl BridgeDefinitionId {
+    /// Default Bridge definition identifier constructor.
+    pub fn new(kind: BridgeKind, name: &str) -> Self {
+        BridgeDefinitionId {
+            kind,
+            name: name.to_owned(),
+        }
+    }
+}
+
+/// A data required for `Bridge` entity initialization.
+#[derive(Encode, Decode, PartialEq, Eq, Debug, Clone, Hash, Io)]
+pub struct BridgeDefinition {
+    /// An Identification of the `BridgeDefinition`.
+    pub id: <BridgeDefinition as Identifiable>::Id,
+    /// Bridge owner's account Identification. Only this account will be able to manipulate the bridge.
+    pub owner_account_id: <Account as Identifiable>::Id,
```

Comment:
This is possible implementation, but maybe we can somehow create a permission for every bridge i runtime. MAybe we can, but it might create additional complexity... maybe in the future...

Reply:
Yes, we can describe it in the RFC.